receiver/hostmetricsreceiver/internal/scraper/processscraper/config.go

```diff
@@ -26,20 +26,65 @@ type Config struct {
 
 	// Metrics allows to customize scraped metrics representation.
 	Metrics metadata.MetricsSettings `mapstructure:"metrics"`
-	// Include specifies a filter on the process names that should be included from the generated metrics.
-	// Exclude specifies a filter on the process names that should be excluded from the generated metrics.
-	// If neither `include` or `exclude` are set, process metrics will be generated for all processes.
-	Include MatchConfig `mapstructure:"include"`
-	Exclude MatchConfig `mapstructure:"exclude"`
```

Comment:
Please provide backward compatibility for this configuration and throw a deprecation warning in logs if it's being used.

Reply:
HI @dmitryax , 
Sorry this PR is still a work in progress.  I have made the requested changes on this PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/8977 (only includes filtering options, not aggregating child metrics).  I accidentally created this PR before the issue associated with it is approved and am not sure how to delete the PR.

Reply:
Np, I can close this PR